css/style.css

```diff
@@ -15,15 +15,16 @@ body {
 }
 
 .header_color_black {
-	display: flex;
-	width: 31.5vw;
-	height: 92px;
+  display: flex;
+	width: 32.5vw;
+  height: 100px;
 	justify-content: space-between;
 	align-items: center;
 	background: black;
 }
 
 .header__title {
+	margin-top: 8px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_A/pullRequest?prid=6206892)